flink-runtime/src/test/java/org/apache/flink/runtime/metrics/dump/QueryScopeInfoTest.java

```diff
@@ -20,58 +20,140 @@
 
 import org.junit.Test;
 
-import static org.apache.flink.runtime.metrics.dump.QueryScopeInfo.INFO_CATEGORY_JM;
```

Comment:
Whether to use static imports or not is currently rather undefined in Flink.

In this case i would remove the static import to make it very clear in the test that we're comparing to a constant that is defined outside the test.